articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -8,31 +8,31 @@ ms.topic: article
+[Get started with the Microsoft Quantum Development Kit](xref:microsoft.quantum.welcome) provides a high-level overview of quantum computing with Q#, including tutorials for writing your first Q# program, including getting started guides and and introduction to the Q# quantum libraries for developing quantum programs.
 
-To guide you in learning quantum computing with Q#, we provide a Getting Started Guide (link below) for the Quantum Development Kit, which provides a journey guide for both those who want to start coding quantum programs and those who are not yet ready to start coding, but want to learn more about Q# and quantum programming.
+## Learning the basics: what do you need to know?
 
-Before embarking on the Getting Started journey, this last article on the “Five things to know about quantum computing” introduces you to some basic concepts that are especially important to understand quantum computing and points to further resources for you along your journey.
+You don’t need to know quantum physics to learn about Q# and quantum computing or start writing quantum applications.
 
-These concepts will give you a good introduction to the fundamental knowledge you need to start coding quantum programs.  Those who are not yet ready to start coding, but want to learn more about Q# and quantum programming can still review these concepts at a high level here and then proceed to the Getting Started Guide to learn more about quantum computing and Q# without having to start coding.
+These concepts will give you a good introduction to the fundamental knowledge you need to start coding quantum programs.  
 
-* [Linear algebra (vectors and matrices)](xref:microsoft.quantum.concepts.vectors): In quantum computing, quantum states are represented by vectors, with quantum operations being linear transformations applied to these vectors.
-* [Complex numbers](https://en.wikipedia.org/wiki/Complex_number): The coefficients of quantum state vectors are complex numbers. You can understand some basic quantum computing concepts without them, but you won't get far before you need to incorporate them into your quantum toolkit.
 * [Basic quantum mechanics](xref:microsoft.quantum.concepts.intro): We just said that you don’t need to know quantum physics to start coding (and it’s true!). But some basic concepts of quantum mechanics and its mathematical notation will be helpful to understand quantum programming.
 
-## Code first, then learn! Or learn more, then code!
+* **Linear algebra (vectors and matrices)**: In quantum computing, quantum states are represented by vectors, with quantum operations being linear transformations applied to these vectors.  Here is a [Jupyter notebook tutorial on Linear Algebra](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/LinearAlgebra).  You can also read more about this in our concept guide about [vectors and matrices](xref:microsoft.quantum.concepts.vectors).
```

Comment:
I apologize for not catching this one earlier, but I would suggest that we avoid the implication that Q# operations are linear transformations in favor of saying that we can simulate Q# operations using linear transformations.
```suggestion
* **Linear algebra (vectors and matrices)**: In quantum computing, quantum states are represented by vectors, with quantum operations being simulated by linear transformations applied to these vectors.  Here is a [Jupyter notebook tutorial on Linear Algebra](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/LinearAlgebra).  You can also read more about this in our concept guide about [vectors and matrices](xref:microsoft.quantum.concepts.vectors).
```

Reply:
I do not see how this is more correct or more clear.  Would you accept "...quantum operations are represented by linear transformations..."

Reply:
We've seen, especially during workshop events, readers and participants thinking that states and unitary operators are objects in the Q# language, rather than internal implementation details to the simulator. My suggestion was intended to try and allay that potential confusion by making a separation between quantum operations and how we represent them inside simulators. That said, I marked as approved as this is only a relatively minor suggestion made to try and forestall something we've learned from the workshops.